### PR TITLE
FOUR-13307:The process table takes at least 10 seconds to load processes

### PR DESCRIPTION
**Pull request by fagubla**
## Issue & Reproduction Steps
1.Log in
2.Go to Designer
3.Click on View all process

**Current Behavior:**

The process loading takes at least 10 seconds, the spinner is lost and shows the blank table for a few seconds

## Solution
- List the changes you've introduced to solve the issue.

## How to Test
Describe how to test that this solution works.

## Related Tickets & Packages
- [FOUR-13307](https://processmaker.atlassian.net/browse/FOUR-13307)

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

ci:deploy
ci:next

[FOUR-13307]: https://processmaker.atlassian.net/browse/FOUR-13307?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by gustavobascope — APPROVED** [review]
No observations

File: resources/js/processes/components/ArchivedProcessList.vue [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+</template>
+
+<script>
+import { createUniqIdsMixin } from "vue-uniq-ids";
+<<<<<<< HEAD
+import datatableMixin from "../../components/common/mixins/datatable";
+import dataLoadingMixin from "../../components/common/mixins/apiDataLoading";
+import isPMQL from "../../modules/isPMQL";
+=======
+>>>>>>> 9ed828451b1a6a04cdefc40bb15eb64bee43d488
```
Comment (nolanpro) [inline resources/js/processes/components/ArchivedProcessList.vue:136]: @fagubla this has merge conflicts comitted

**Review by nolanpro — CHANGES_REQUESTED** [review]

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-a25098fa53.engk8s.processmaker.net

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-a25098fa53.engk8s.processmaker.net

**Pull request merged**